tests/integration_tests/viz_tests.py

```diff
@@ -739,19 +739,13 @@ def test_get_data_empty_null_keys(self):
                     "group": "All",
                 }
             ],
-            "NULL": [
-                {
-                    "values": [
-                        {"x": 100, "y": 10},
-                        {"x": 200, "y": 20},
-                        {"x": 300, "y": 30},
-                    ],
-                    "group": "All",
-                }
-            ],
         }
         self.assertEqual(data, expected)
 
+        form_data = {"groupby": [], "metrics": [None]}
```

Comment:
Should we add a test case where metric is a dict, too?

Reply:
the more unit test at [here](https://github.com/apache/superset/blob/5572f3f42442e617d55759e7203d590102adc58b/tests/unit_tests/core_tests.py#L89). I added some invalid metric case as well.